CONTRIBUTING.md

```diff
@@ -8,12 +7,15 @@ You can contribute by:
 * Improving documentation
 * Talking about Strimzi
 
-All bugs, tasks or enhancements are tracked as [GitHub issues][issues].
-Issues which might be a good start for new contributors are marked with the [“good-start”][newbie-issues] label.
+All bugs, tasks or enhancements are tracked as [GitHub issues][issues]. Issues which might be a good start for new contributors are marked with the [“good-start”][newbie-issues] label.
+
+The [developer quick start guide][dev-quick-start] describes how to quickly get setup to build Strimzi from source.
+
+The [hacking guide][hacking-guide] describes in more detail the various ways you can build Strimzi and how to test your changes before submitting a patch or opening a pull request.
 
-The [Hacking guide][hacking-guide] describes how to build Strimzi and how to test your changes before submitting a patch or opening a PR.
+The [release checklist][release-list] describes the steps that need to be completed for a new version release.
```

Comment:
two blank lines?

Reply:
Do you mean between each paragraph?

Reply:
Between the above line and the following one, or am I wrong? Maybe yes :-)

Reply:
There is one line empty line between each paragraph in the source

Reply:
Ok so I am blind ;)

Reply:
No on here it looks like there isn't a line but in the source there is?